posthog/settings/web.py

```diff
@@ -241,7 +241,7 @@
 LOGOUT_URL = "/logout"
 LOGIN_REDIRECT_URL = "/"
 APPEND_SLASH = False
-CORS_URLS_REGEX = r"^/api/(?!early_access_features|surveys).*$"
+CORS_URLS_REGEX = r"^/api/(?!early_access_features).*$"
```

Comment:
If this is wrong then it must also be wrong for early access features right? So why only removing surveys? Can someone in the know look at this and figure out if any of this even is used...